lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -719,6 +720,9 @@ def setUp(self):
         )
         self.team.add_user(self.user)  # pylint: disable=no-member
 
+        # Create the user a UserProfile so it doesn't act as a direct access user
+        UserProfile(user=self.user).save()
```

Comment:
Actually, I take the opposite view. If something changes with direct access, or being registered, then I'd like the test to fail again so I can determine why. It may be an early indicator that direct access is broken again.